setup.py

```diff
@@ -32,6 +32,7 @@
     "ray[debug,tune]~=0.8.5",
     "scipy>=1.8.0",
     "wandb",
+    "huggingface_sb3",
```

Comment:
careful, need to wait for new release, last release still has the annoying `pickle5` dependency

Reply:
I think this is fixed by now.